src/main/java/uk/gov/hmcts/ccd/domain/model/std/CaseAssignedUserRole.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.ccd.domain.model.std;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+public class CaseAssignedUserRole {
+
+    @JsonProperty("case_data_id")
```

Comment:
`case_id` is used in LLD and other references

Reply:
Done